examples/06_manipulating_images/plot_threshold_image.py

```diff
@@ -0,0 +1,187 @@
+# %%
+"""
+Image thresholding
+==================
+
+The goal of this example is to illustrate the use of the function
+:func:`~nilearn.image.threshold_img` together with ``threshold`` and
+``two_sided`` parameters to view certain values in image data.
+
+The ``threshold`` parameter can take both positive and negative values.
+``two_sided`` parameter is complementary to ``threshold`` effecting its
+behavior.
+
+"""
+
+# %%
+# Image without threshold
+# -----------------------
+#
+# We will first load the dataset and display the image without manipulation.
+
+from nilearn import datasets, plotting
+
+cmap = "RdBu_r"
```

Comment:
```suggestion
cmap = "RdBu_r"
```
this is already the default cmap for plot_stat_map so I don't think we need to declare / pass it as a variable: let's only change it where we need a sequential colormap